### PR TITLE
Refactor pickers

### PR DESCRIPTION
**Pull request by irisli**
This PR is a refactor of the Pickers driven by data and the PickerGenerator. This replaces it with a more code-based approach to it.

This also makes pages in the Laboratory persistent between app page change. https://github.com/stellar/laboratory/issues/44

### CODE REVIEW

File: src/actions/endpointExplorer.js [hunk 1]
```diff
@@ -17,11 +17,11 @@ export function changePendingRequestProps(props) {
 }
 
 export const UPDATE_VALUES = "UPDATE_VALUES";
-export function updateValues(param, values) {
```
Comment (nullstyle) [inline src/actions/endpointExplorer.js]: If this action is becoming single-valued, you should reflect it in the name as well:  UPDATE_VALUE

Comment (irisli) [inline src/actions/endpointExplorer.js]: Thanks

**Comment by nullstyle** [comment]
You've copy-pasted the same markup for a "text box with validation" too many times for me to endorse this change.

Instead, I think you should add `TextPicker` which all of these classes (`PubKeyPicker`, `SecretKeyPicker`, `PositiveIntPicker`, `AmountPicker`, etc.) use instead of repeating code.  Their only differences are the "what validator to use" and "what placeholder to use", as far as I can see.  So, for example:

```js
export default function SecretKeyPicker(props) {
  return <TextPicker 
    {...props} 
    placeholder={'Example: SAEXAMPLE6TLGEF6ASOTVTLFUK7LE2K2PFVPFGTEZMMVHH7KLLBBROEQ'}
    validator={validator}
    />
}
```

**Comment by nullstyle** [comment]
Other than the copy-pasta, +1

**Comment by irisli** [comment]
Thanks for the useful feedback!

**Pull request merged**